src/libs/ui/docsetsdialog.cpp

```diff
@@ -59,6 +59,13 @@ const char ApiServerUrl[] = "http://api.zealdocs.org/v1";
 const char RedirectServerUrl[] = "https://go.zealdocs.org";
 // TODO: Each source plugin should have its own cache
 const char DocsetListCacheFileName[] = "com.kapeli.json";
+const char UserContributedDocsetListCacheFileName[] = "com.kapeli.json.user_contributed";
+
+// TODO manage mirror
+const char UserContributedApiUrl[] =
+  "http://london.kapeli.com/feeds/zzz/user_contributed/build/index.json";
+const char UserContributedDocsetListUrl[] =
+  "http://london.kapeli.com/feeds/zzz/user_contributed/build/%1/%2";
```

Comment:
Maybe we should use `http://kapeli.com/feeds/zzz/user_contributed/build/index.json` and `http://kapeli.com/feeds/zzz/user_contributed/build/%1/%2`. for better CDN usage.

Reply:
"kapeli.com" just goes to the server that hosts my website, which is in the same datacenter as newyork.kapeli.com. I prefer if you don't use kapeli.com but one of the other mirrors.

Reply:
I see, I had expected "kapeli.com" has geo-based DNS solver, pointing to mirrors in other regions. Thanks for the clarification.